ksql-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -62,8 +62,8 @@ statement
     | CREATE TABLE (IF NOT EXISTS)? qualifiedName
             (WITH tableProperties)? AS query                                #createTableAs
     | DROP TOPIC (IF EXISTS)? qualifiedName                                 #dropTopic
-    | DROP STREAM (IF EXISTS)? qualifiedName                                #dropStream
-    | DROP TABLE (IF EXISTS)? qualifiedName                                 #dropTable
+    | DROP STREAM (IF EXISTS)? qualifiedName (WITH TOPIC)?                  #dropStream
```

Comment:
I wonder if `WITH` is right here? The topic already gets dropped from the metastore. Perhaps it should be `DELETE`?

Reply:
We have KSQL topic and kafka topic. You are right, the KSQL topic is dropped from the metastore but this is for Kafka topic. Can you elaborate on how using `DELETE` would look like?